tests/test_from_thread.py

```diff
@@ -63,6 +67,23 @@ def thread_worker_sync(func: Callable[..., T_Retval], *args: Any) -> T_Retval:
     return from_thread.run_sync(func, *args)
 
 
+# Duplicate anyio_backend for BlockingPortal tests that need to be parametrized with
+# pairs of backends.
+portal_backend = anyio_backend
+
+
+@pytest.fixture
+def portal_backend_name(portal_backend: Any) -> str:
+    return anyio_backend_name.__wrapped__(portal_backend)  # type: ignore[attr-defined]
+
+
+@pytest.fixture
+def portal_backend_options(portal_backend: Any) -> dict[str, Any]:
+    return anyio_backend_options.__wrapped__(  # type: ignore[attr-defined]
+        portal_backend
+    )
```

Comment:
Do the tests specifically need to replicate the exact options for each backend? If not, you could just parametrize the tests based on `anyio.get_all_backends()`.